doc/mgr/iostat.rst

```diff
@@ -3,30 +3,43 @@
 iostat
 ======
 
-This module shows the current throughput and IOPS done on the Ceph cluster.
+The ``iostat`` module shows the current throughput and IOPS done on the Ceph
```

Comment:
The ``iostat`` module reports metrics for cluster throughput and IOPS.

Reply:
I've added this in, but I failed to make a note of it in my handwritten ledger. Anyway, this suggestion is incorporated into the docs.